packages/sync-service/lib/electric/telemetry/open_telemetry.ex

```diff
@@ -179,9 +179,9 @@ defmodule Electric.Telemetry.OpenTelemetry do
   e.g.
 
   ```elixir
-  OpenTelemetry.start_interval("quick_sleep")
+  OpenTelemetry.start_interval(:quick_sleep)
```

Comment:
```suggestion
  OpenTelemetry.start_interval(:"quick_sleep.duration_µs")
```
just so the comment below it makes sense and to show best practices